app/models/ability.rb

```diff
@@ -72,6 +72,12 @@ def initialize(user)
         can :access, :alaveteli_pro
       end
 
+      # Creating embargoes
+      can :create_embargo, InfoRequest do |info_request|
+        user && info_request.user.is_pro? && !info_request.embargo &&
+          (user == info_request.user || user.is_pro_admin?)
```

Comment:
Could move checking for whether or not `info_request_batch_id.nil?` here. Would remove need to limit the create form remove being outputted and remove the check in the controller action too.

Reply:
have done - I _think_ we can do that without breaking a future requirement to update batches as that will be for `can :create_embargo, InfoRequestBatch`. I think.